packages/substrate-bindings/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 - Update dependencies
 
+- Incorrect bytes to skip when decoding storage key
+
```

Comment:
It's under 0.6.2 :) Move it under `## Unreleased` please 🙏

Reply:
oh shoot :))

Reply:
Classic case of ✋ faster than 🧠, have moved this to "unreleased".